alembic/versions/2024_09_04_18dbadd8c436_rename_fastq_to_raw_data.py

```diff
@@ -0,0 +1,127 @@
+"""rename_fastq_to_raw_data
+
+Revision ID: 18dbadd8c436
+Revises: bb4c6dbad991
+Create Date: 2024-09-04 13:15:11.876822
+
+"""
+
+from enum import StrEnum
+
+from sqlalchemy import orm, types
+from sqlalchemy.dialects import mysql
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "18dbadd8c436"
+down_revision = "bb4c6dbad991"
+branch_labels = None
+depends_on = None
+
+
+base_options = (
+    "balsamic",
+    "balsamic-pon",
+    "balsamic-qc",
+    "balsamic-umi",
+    "demultiplex",
+    "fluffy",
+    "jasen",
+    "microsalt",
+    "mip-dna",
+    "mip-rna",
+    "mutant",
+    "raredisease",
+    "rnafusion",
+    "rsync",
+    "spring",
+    "taxprofiler",
+    "tomte",
+)
+old_options = sorted(base_options + ("fastq",))
+new_options = sorted(base_options + ("raw-data",))
+
+old_enum = mysql.ENUM(*old_options)
+new_enum = mysql.ENUM(*new_options)
+
+
+class Workflow(StrEnum):
+    BALSAMIC: str = "balsamic"
+    BALSAMIC_PON: str = "balsamic-pon"
+    BALSAMIC_QC: str = "balsamic-qc"
+    BALSAMIC_UMI: str = "balsamic-umi"
+    DEMULTIPLEX: str = "demultiplex"
+    FLUFFY: str = "fluffy"
+    JASEN: str = "jasen"
+    MICROSALT: str = "microsalt"
+    MIP_DNA: str = "mip-dna"
+    MIP_RNA: str = "mip-rna"
+    MUTANT: str = "mutant"
+    RAREDISEASE: str = "raredisease"
+    RAW_DATA: str = "raw-data"
+    RNAFUSION: str = "rnafusion"
+    RSYNC: str = "rsync"
+    SPRING: str = "spring"
+    TAXPROFILER: str = "taxprofiler"
+    TOMTE: str = "tomte"
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class Case(Base):
+    __tablename__ = "case"
+    id: Mapped[int] = mapped_column(primary_key=True)
+    data_analysis: Mapped[str | None]
+
+
+class Analysis(Base):
+    __tablename__ = "analysis"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    workflow: Mapped[str | None]
+
+
+def upgrade():
```

Comment:
Noticed when troubleshooting the revisions - might be nice to index the columns case.data_analysis and analysis.workflow? Something like 

```Python
    op.create_index("case_workflow_index", "case", ["data_analysis"], unique=False)
```